app/2.0/docs/tools/polymer-cli.md

```diff
@@ -26,6 +31,10 @@ title: Polymer CLI
 
         npm install -g polymer-cli
 
+    In the output from this command, you may see an npm warning about Bower being deprecated. You 
```

Comment:
Oh, shoot.I merged this and I missed that this note doesn't have a run-in head. Should be something like "**Bower deprecation warning.**" Or perhaps if you're feeling waggish, **Please ignore the warning.**